src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,74 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
 public class WorkWithFile {
+    private static final String FILE_SEPARATOR = ",";
+    private static final String SUPPLY_OPERATION = "supply";
+    private static final String BUYING_OPERATION = "buy";
+    private static final int THEME_INDEX = 0;
+    private static final int VALUE_INDEX = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(fromFileName);
+        StringBuilder stringBuilder = new StringBuilder();
+        int buy = 0;
+        int supply = 0;
+        try (BufferedReader reader = new BufferedReader(new FileReader(file))) {
+            String line = reader.readLine();
+            while (line != null && !line.isEmpty()) {
+                stringBuilder.append(line).append(System.lineSeparator());
+                line = reader.readLine();
+            }
+            String[] operationsWithGoods = stringBuilder.toString().split(System.lineSeparator());
+            for (String operationWithGood : operationsWithGoods) {
```

Comment:
This loop is redundant. You already have a line in 26 line, just continue processing this line in the same loop.
After removing the loop, check which variables can be removed also.